docs/UserGuide.md

```diff
@@ -112,6 +112,17 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
+### Editing a tag : `tag edit`
+
+Edits an existing tag in the address book.
+
+Format: `edit tag INDEX t/tag`
```

Comment:
i think it should be `tag edit ...` instead of  `edit tag...`?

Reply:
yep done thanks!!!